README.md

```diff
@@ -103,9 +103,9 @@ Where possible we stick to Rails defaults.
       `spec/features/accessibility`.
 - Front-end
   - Rename `app/javascript` to `app/frontend`
-  - Setup Webpacker.
+  - Setup Shakapacker (the maintained fork of Webpacker).
   - > **Note** We are trialing the new JS packaging options that Rails 7+
-    > provides. For now our default is still Webpacker because it provides us
+    > provides. For now our default is still Shakapacker because it provides us
```

Comment:
@G-Rath I missed in my review of your shakapacker PR that we should update the README. It's a trivial change so I'm just doing it here rather than a separate PR.